src/server/system_services/bucket_server.js

```diff
@@ -1319,6 +1291,35 @@ async function claim_bucket(req) {
         }
     }
 
+    try {
```

Comment:
I think we can delete claim_bucket - we don't use it to the best of my knowledge

Reply:
I will once find references to this function and if found none, I will remove it ✅ .